honeycombio/data_source_column_test.go

```diff
@@ -44,8 +44,8 @@ func TestAccDataSourceHoneycombioColumn_basic(t *testing.T) {
 	})
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:          testAccPreCheck(t),
-		ProviderFactories: testAccProviderFactories,
+		PreCheck:                 testAccPreCheck(t),
+		ProtoV5ProviderFactories: testAccProtoV5ProviderFactory,
```

Comment:
All of these updates in the SDK-based tests were needed eventually as they need to be given a provider factory which can access the Framework-based resources/datasources/